core/domain/html_cleaner.py

```diff
@@ -790,3 +800,57 @@ def add_caption_attr_to_image(html_string):
             image['caption-with-value'] = escape_html(json.dumps(''))
 
     return unicode(soup)
+
+
+def add_dimensions_to_noninteractive_image_tag(html_string, exp_id):
+    """Adds dimensions to all oppia-noninteractive-image tags.
+
+    Args:
+        html_string: str. HTML string in which the dimensions is to be
+            added.
+        exp_id: str. Exploration id.
+
+    Returns:
+        str. Updated HTML string with the dimensions for all
+            oppia-noninteractive-image tags.
+    """
+    soup = bs4.BeautifulSoup(html_string.encode('utf-8'), 'html.parser')
+
+    for image in soup.findAll(name='oppia-noninteractive-image'):
+        filename = unescape_html(image['filepath-with-value'])
+        filename = filename[1:-1]
+        image['filepath-with-value'] = escape_html(json.dumps(
+            get_filepath_of_object_image(filename, exp_id)))
+    return unicode(soup)
+
+
+def get_filepath_of_object_image(filename, exp_id):
+    """Gets the dimensions of the image file.
```

Comment:
Change to " Gets an object containing the filename, width and height of the filename provided"

Reply:
Yes, sorry I misinterpreted the comment. I wrote it for later changes, I will change the description of the function as written. Let the travis result come and then I'll change this. Thanks.